ingest/scripts/call_loculus.py

```diff
@@ -218,21 +218,36 @@ def regroup_and_revoke(metadata, sequences, map, config: Config, group_id):
     Submit segments in new sequence groups and revoke segments in old (incorrect) groups in Loculus.
     """
     response = submit_or_revise(metadata, sequences, config, group_id, mode="submit")
-    new_accessions = response[0]["accession"]  # Will be later added as version comment
-
-    url = f"{organism_url(config)}/revoke"
+    new_accessions = {}  # Map from submissionId to new loculus accession
+    for item in response:
+        new_accessions[item["submissionId"]] = item["accession"]
 
     to_revoke = json.load(open(map, encoding="utf-8"))
 
-    loc_values = {loc for seq in to_revoke.values() for loc in seq.keys()}
-    loculus_accessions = set(loc_values)
-
-    accessions = {"accessions": list(loculus_accessions)}
+    old_loculus_keys: dict[
```

Comment:
(optional) Above you define `new_accessions` where the values are new accessions, here the values are also `new_accessions` but the name is `old_loculus` (I appreciate the name does contains "keys"!) For me this would be clearer as `old_to_new` and the above as `submission_id_to_new` or something.